sources/packages/backend/apps/api/src/services/student-assessment/student-assessment.service.ts

```diff
@@ -91,6 +92,62 @@ export class StudentAssessmentService extends RecordDataModelService<StudentAsse
     return query.getOne();
   }
 
+  /**
+   * Get the pending assessment for the institutions which have integration true for the particular date.
+   * @param generatedDate Date in which the previous days records has to be fetched.
```

Comment:
The comment is not accurate. The `generatedDate` is the date to generate the file, when it is not provided it will use the previous date as the default value.